### ISSUE TITLE
Java icon (and potentially others) not horizontally centering properly.

### ISSUE THREAD

**Issue by KennyWibowo** [fizzed/font-mfizz#48]
Using text-align: center really brings this out.

![image](https://cloud.githubusercontent.com/assets/8093099/18116378/29e7343a-6efc-11e6-8971-b77d4156b010.png)

![image](https://cloud.githubusercontent.com/assets/8093099/18116342/f04dd1d4-6efb-11e6-9b96-9115122bf09d.png)

**Comment by KennyWibowo** [comment]
Could potentially add to the CSS properties to fit the box around the icon exactly, though not sure, not much of a CSS wizard.

**Comment by tiaaaa123** [comment]
I'm having the same issue right now. With multiple icon.

**Comment by xied75** [comment]
This really is an issue, if I drag and drop "java-bold" svg file onto http://fontello.com/, this java icon will push all my other well-behaved icons up and being clipped. Anyone knows how to fix this?

**Comment by jjlauer** [comment]
@xied75 If I had to guess its probably something with the svg bounding box.  If you open up the svg in an editor that's something that can be tweaked.  When I add icons I was careful about tweaking/simplifying the svgs to be "renderable" to a font.  Not every svg can easily be converted so you need to do a number of things.

Now, that being said, in the many years after this font's release, I bet there are better tools we could compile the font with compared to when it was originally developed.  I haven't had time recently to try and give some of the new tools a shot, but maybe I can try to find some time in the next couple months.  If anyone in the community could give it a try, any PRs would be awesome!

**Comment by Alhadis** [comment]
> If I had to guess its probably something with the svg bounding box

Yep, it is. Actually, most of the icons are misaligned or sized incorrectly, which is what led to me [fixing them all by hand](https://github.com/file-icons/MFixx). Didn't consider a PR because the adjustments would've been a breaking change for some layouts, I imagine.

So yeah, easy fix for now is to [use this SVG](https://github.com/file-icons/MFixx/blob/master/svg/java-bold.svg) for the `java-bold` icon. =)

**Comment by jjlauer** [comment]
@Alhadis Yeah, I can sympathize with the hand work.  It was a lot of work to put together what's in this repo too.  I actually do think those as a PR would have been awesome, but I get why you had concerns about breaking compat.  I read your docs on optimizing geometry and I'm wondering what editor/tool you ended up using?

**Comment by Alhadis** [comment]
Just Adobe Illustrator, and having an attention to detail. That's seriously all. =)

**Comment by Alhadis** [comment]
I have a background in graphic design, so I may as well put it to use. 😀

**Comment by jjlauer** [comment]
@Alhadis I've got almost zero graphic design experience -- so its cool to see how you did it :-)  I tried using only open source tools, but I'm guessing Adobe is the only way to really go in the end.  I'll dig into what you did further to see how we can polish things up.

**Comment by Alhadis** [comment]
Hahah! Nooo, the choice of program has nothing to do with it... I've been using Adobe software for over 10 years, so everything's burned deeply into muscle memory. What I did could've easily been done by somebody with an equal level of skill in Inkscape. =)
